apps/pdc-frontend/src/middleware.ts

```diff
@@ -15,21 +15,52 @@ const getOpenFormsHost = () => {
   return createOpenFormsApiUrl()?.host || '';
 };
 
-const cspDevelopmentHeader = () =>
-  `default-src 'self';
-    script-src 'self' siteimproveanalytics.com ${getOpenFormsHost()} 'unsafe-inline' 'unsafe-eval';
-    style-src 'self' localhost:8000 'unsafe-inline';
-    connect-src 'self' ${getOpenFormsHost()};
-    img-src 'self' blob: data:;
-    font-src 'self';
-    object-src 'none';
-    base-uri 'self';
-    form-action 'self';
-    frame-ancestors 'none';
-    block-all-mixed-content;`;
+const { cspDevelopmentHeader, nonce, EVAL, INLINE, SELF, NONE, BLOB, DATA } = require('csp-header');
+
+cspDevelopmentHeader({
```

Comment:
`cspDevelopmentHeader` does not exist in the  `'csp-header'` package

Reply:
It did not work on the development 

<img width="961" alt="Screenshot 2024-03-06 at 15 04 20" src="https://github.com/frameless/strapi/assets/53454241/e485a12d-3eee-459f-a049-e809eb469f62">